BUILDTARGETS.md

```diff
@@ -2,24 +2,19 @@
 
 The Unity Buy SDK currently supports the following build targets:
 
+* [Android](#Android)
 * [iOS](#ios)
 
-# iOS
-
-### Requirements
-
-The Unity Buy SDK requires iOS games to have a minimum target SDK of iOS 9 and support Swift 3.
+You can find more esoteric details regarding platform specific dependencies and the build phase in [Build Details](BUILDDETAILS.md)
```

Comment:
Can we switch `esoteric details` to `more detailed explanation`. I think this would be more inline with Shopify copy.